src/arch/xtensa/configs/tgph_ipc4_defconfig

```diff
@@ -0,0 +1,13 @@
+CONFIG_TIGERLAKE=y
+CONFIG_RIMAGE_SIGNING_SCHEMA="tgl-cavs"
+CONFIG_CAVS_IPC4=y
```

Comment:
`CONFIG_IPC_MAJOR_4=y`?

we can also make change in xtensa-build-all.sh like this, not sure which is better. https://github.com/aiChaoSONG/sof/commit/b9754728ffec70b06434607309a7f2ae28eb0e0e

Reply:
sure, let's do it in script